v1/santoni.json

```diff
@@ -71,12 +71,12 @@
           "group": "firmware",
           "files": [
             {
-              "url": "https://ci.ubports.com/view/Device%20Compatibility%20Images/job/Device%20Compatibility%20Images/job/halium-santoni/184/artifact/halium-unlocked-recovery_santoni.img",
-              "checksum": "8235406a210804e65b591b4f00728b574396c67f78f90134916debf3b681039f"
+              "url": "https://ci.ubports.com/view/Device%20Compatibility%20Images/job/Device%20Compatibility%20Images/job/halium-santoni/196/artifact/halium-unlocked-recovery_santoni.img",
+              "checksum": "25798584952f29273d3ec2ebf63e7551db26f48e447493461627172330d616a5"
             },
             {
-              "url": "https://ci.ubports.com/view/Device%20Compatibility%20Images/job/Device%20Compatibility%20Images/job/halium-santoni/184/artifact/halium-boot_santoni.img",
-              "checksum": "0845be37fff8d3c376f3c9d6e4f54632c66921571b129194150b94689beb5b54"
+              "url": "https://ci.ubports.com/view/Device%20Compatibility%20Images/job/Device%20Compatibility%20Images/job/halium-santoni/196/artifact/halium-boot_santoni.img",
```

Comment:
Why does it attempt to fetch halium-boot as part of firmware group? It is not needed to boot into recovery and recovery should flash a boot image from OTA package instead anyway.

Reply:
Some devices are weird like that and require you to also flash boot before recovery is bootable. Dunno if this is actually the case for santoni.

Reply:
I'm only updating what was already there, as the installer currently cannot install on santoni.